app/managers.py

```diff
@@ -0,0 +1,59 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str,
+    ) -> None:
+        self._connection.execute(
+            f"INSERT INTO  {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()

```

Comment:
It's a good practice to handle exceptions that may occur during database operations, such as connection errors or SQL syntax errors, to make the code more robust.